spec/plurimath/mathml/parser_spec.rb

```diff
@@ -1089,6 +1089,45 @@
     end
   end
 
+  context "contains mathml string of dot decimal values only" do
+    let(:exp) {
+      <<~MATHML
+        <math xmlns='http://www.w3.org/1998/Math/MathML'>
+          <msgroup>
+            <mn>1.4</mn>
+            <mn>1.4.5</mn>
```

Comment:
`1.4.5` is not a number, it should not be accepted.

Please refer to https://developer.mozilla.org/en-US/docs/Web/MathML/Element/mn for the accepted values.